my_portfolio.html

```diff
@@ -1,13 +1,63 @@
 ﻿<!DOCTYPE html>
-
 <html lang="en">
+
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
 </head>
+
 <body>
-    <h1>Hello world</h1>
+    <img src="icons/header-illustration-mobile.svg" id="header-img" />
+    <img src="icons/header-desktop.svg" id="header-img-desktop" />
+    <div id="nav-bar-ctn">
+        <div id="nav-bar">
+            <a>
+                <p>Portfolio</p>
+            </a>
+            <a>
+                <p>About</p>
+            </a>
+            <a>
+                <p>Contact</p>
+            </a>
+        </div>
+    </div>
+    <div id="toolbar">
+        <div id="menu-button">
+            <object data="icons/icon-menu.svg"></object>
+        </div>
+    </div>
+
+    <div id="headline">
+
+        <div id="primary-text">
+            <p id="title-1">Hey there. I'm Gumaro.</p>
+            <p id="title-2">I'm a software developer</p>
+        </div>
```

Comment:
- [ ] Kindly consider adding this in a h1  as its the main header of the page because If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more infoIf you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info